docsite/docs/Docs/fal-cli/python-models.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 2
+---
+
+# Making Python dbt models
+
+> ⚠️ NOTE: This feature is enabled with the flag `--experimental-models`
+
+There are some cases where a SQL transformation is either too complex or impossible. With `fal flow` you can build your dbt models in Python.
```

Comment:
As of version 0.2.3, fal now supports a new building block: "Pure Python models". This new type of model allows you to represent a dbt model purely in Python code leveraging its rich ecosystem of libraries. With Python models, you can now go beyond the capabilities of SQL and build:

1) Data transformations that Python is better suited for e.g. text manipulation, leveraging external libraries.  
2) Use Data Science libraries such as sklearn, xgboost to represent ML model artifacts such as predictions as dbt models.